src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1753,14 +1753,14 @@ public JObject CreateJObject<T>(T value, string type, string description, bool w
             return ret;
 
         }
-        public JObject CreateJObjectForBoolean(int value)
+        public JObject CreateJObjectForBoolean(bool value)
         {
-            return CreateJObject<bool>(value == 0 ? false : true, "boolean", value == 0 ? "false" : "true", true);
+            return CreateJObject<bool>(value, "boolean", value.ToString().ToLower(), true, "System.Boolean");
         }
 
         public JObject CreateJObjectForNumber<T>(T value)
         {
-            return CreateJObject<T>(value, "number", value.ToString(), true);
+            return CreateJObject<T>(value, "number", Convert.ToString(value, System.Globalization.CultureInfo.InvariantCulture), true);
```

Comment:
AssignmentTests.InspectVariableBeforeAndAfterAssignment for MONO_TYPE_R4 and MONO_TYPE_R8 on Windows were failing and this change fixes it, at least for me locally. There was a comma because of my local settings. We need a result that is culture invariant.

Reply:
Add that to the other PR as a separate commit with the explanation in the commit message. In general, you want to make sure that you can track the reasoning behind changes. And it helps anyone reviewing, to check it too :)

Reply:
https://github.com/dotnet/runtime/pull/66979